x-pack/plugins/index_lifecycle_management/public/application/services/policies/delete_phase.ts

```diff
@@ -57,7 +57,7 @@ export const deletePhaseToES = (
     esPhase.min_age = `${phase.selectedMinimumAge}${phase.selectedMinimumAgeUnits}`;
   }
 
-  esPhase.actions = esPhase.actions ? { ...esPhase.actions } : {};
+  esPhase.actions = esPhase.actions ? { ...esPhase.actions, delete: {} } : { delete: {} };
```

Comment:
```suggestion
  esPhase.actions = esPhase.actions ? {  delete: {}, ...esPhase.actions } : { delete: {} };
```

Since the `delete` action can have a value set in it and we don't want to override it with an empty object (https://www.elastic.co/guide/en/elasticsearch/reference/current/ilm-delete.html#ilm-delete-options)

Reply:
Thanks for spotting this, @jloleysens! You're right and the test agrees with you :)